### DISCUSSION TITLE
Support Firefox ESR version

### DISCUSSION THREAD

**Discussion by mrx23dot** [Authenticator-Extension/Authenticator#1268] [Ideas]
Addon site 
https://addons.mozilla.org/en-US/firefox/addon/auth-helper/?src=external-website
won't allow me to install the addon to 

Firefox v115 ESR (last long term version for non Windows 10)

any chance we could make it compatible with it, by making the needed function not working? 
(I guess it's the screen capturing, which is not essential since we have manual entry)

**Comment by mymindstorm** [comment]
115 is EOL in about a month. The current ESR release (128) is supported by the extension.

https://support.mozilla.org/en-US/kb/firefox-enterprise-115-release-notes

**Comment by mrx23dot** [comment]
Yeah but 128 drops support for Windows 7-8.1 and macOS 10.12-10.14

**Reply by mymindstorm** [reply]
I would strongly recommend against using a browser or OS version that isn't getting security updates. We don't plan on making any changes to support a soon to be unsupported browser version.

You are free to fork the extension.
